Kafka start instructions

oleg-rakitin opened the pull request.

oleg-rakitin committed:
Added instructions for running mongoose to test Kafka

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Oleg Rakitin limonchik16@gmail.com

oleg-rakitin committed:
Fix links

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Oleg Rakitin limonchik16@gmail.com

oleg-rakitin committed:
Fix version

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Oleg Rakitin limonchik16@gmail.com

akurilov commented on README.md:
````diff
@@ -160,4 +160,13 @@ Computer configuration:
++ Take the last preempt dependency ([http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-preempt/](http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-preempt/)) for Kafka storage driver and put in .mongoose\version_of_mongoose\ext
++ Build Kafka storage driver and put in .mongoose\version_of_mongoose\ext
++ Deploy Apache Kafka
++ Run Mongoose ```java -jar mongoose-base-x.x.xx.jar - store-driver-type = kafka ...```
````
```
--storage-driver-type=kafka \
--storage-driver-limit-queue-input=10000 \
--load-batch-size=100 \
...
```

akurilov commented on README.md:
```diff
@@ -160,4 +160,13 @@ Computer configuration:
+
++ Take the last mongoose-base-x.x.xx.jar ([http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-base/](http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-base/))
++ Take the last preempt dependency ([http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-preempt/](http://repo1.maven.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-preempt/)) for Kafka storage driver and put in .mongoose\version_of_mongoose\ext
++ Build Kafka storage driver and put in .mongoose\version_of_mongoose\ext
```
It's important that `.mongoose` directory is in the home directory. I usually refer this as `~/.mongoose`.

akurilov reviewed (changes requested):
see the previous comments

oleg-rakitin committed:
Fix way up .mongoose, fix the startup string example

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Oleg Rakitin limonchik16@gmail.com

oleg-rakitin committed:
Fix code block

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Oleg Rakitin limonchik16@gmail.com

oleg-rakitin committed:
Fix code block

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Oleg Rakitin limonchik16@gmail.com

oleg-rakitin committed:
Fix slashes in path to ext

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Oleg Rakitin limonchik16@gmail.com
